ratis-server/src/test/java/org/apache/ratis/TestReConfigProperty.java

```diff
@@ -196,13 +197,13 @@ public void testReconfigure() {
         dummy.reconfigureProperty(PROP1, null);
         dummy.startReconfiguration();
         RaftTestUtil.waitFor(() -> dummy.getReconfigurationStatus().ended(), 100, 60000);
-        Assert.assertEquals(PROP1 + "set to wrong value ", DEFAULT,
-            dummy.getProperties().get(PROP1));
+        Assertions.assertEquals(DEFAULT
+,             dummy.getProperties().get(PROP1), PROP1 + "set to wrong value ");
       } catch (ReconfigurationException | IOException | InterruptedException | TimeoutException e) {
         exceptionCaught = true;
       }
-      Assert.assertFalse("received unexpected exception",
-          exceptionCaught);
+      Assertions.assertFalse(
+         exceptionCaught, "received unexpected exception");
```

Comment:
Let's use `fail(..)` inside the catch-block instead of `assertFalse(..)`.

Reply:
@szetszwo Thank you very much for reviewing this PR and providing suggestions! I will continue to improve this PR.

Reply:
I need some time to validate the unit tests for the issue and will provide a response as soon as possible.